src/app/layout/spinner/spinner.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="overlay" *ngIf="(spinnerService.showSpinner$ | async) == true">
+  <div class="govuk-box spinner-wrapper" aria-busy="true">
+    <div id="loader" class="govuk-c-loader" role="progressbar" aria-label="Loading, please wait." tabindex="0">
+      <div class="spinner-container">
+        <div class="spinner">
+          <div></div>
```

Comment:
What are all these divs for ?

Reply:
Came from the old implementation, so unsure but just copied it as is.